src/3/world1.md

```diff
@@ -410,178 +411,180 @@ My body, which has no unnecessary meat, is in fact the best!"
-Once activated, each of these layers could act as powerful protective barrier.
+Reiren, who finished changing, was now in her seven layered silk garments.
+It was an elven magic tool.
+Once activated, each of the layers could act as powerful protective barriers.
 
-"I'm going to accompany thee as representative of foreign gods.
+"I'm going to accompany thee as a representative of foreign gods.
 And in order to avenge Alfreyja-dono we're going to attack western country, where cryptids are.
 Right?"
 
 "Yeah, but..."
 
-Looking back at the elf, who was looking up at her, female commander of Urza Resistance answered with all seriousness.
+Looking back at the elf, who was looking up at her, the female commander of Urza Resistance answered with all seriousness.
 
 "We're heading to the South, not the West.
-We're heading to Yurun country on the South, which is now Spirits domain."
+We're heading to Yurun country on the South, which is currently the domain of the spirits."
 
 "Wha!?"
 
-It was completely different story for her.
-But before shrine maiden could rise her objection, Kai continued after Jeanne.
+It was a completely different story for her.
+But before shrine maiden could raise objections, Kai continued after Jeanne.
 
 "That's why I came here looking for you.
 To let you know what elven elder said."
 
 "I heard nothing of it though?"
 
-"Someone was enjoying her bath, while we discussed it.
-And we already got it decided."
+"Someone was enjoying her bath, while we were discussing it.
+And it already got decided."
 
 "...Mu. That's... Well..."
 
 "I'll tell you the reason on our way, let us return quickly."
 
-He gave a silent cue to Rinne, who finished changing.
+He gave a silent cue to Rinne, who had finished changing.
 Kai pointed towards the animal trail, that led to the village, and walked deep into the ancient forest.
 
 -------------------------------------------------------------------------------
 
 Io Federation...
-It is a big country in eastern part of the continent and at the moment being under control of foreign gods.
-Humans, who lost in the war, had to reconstruct ruins into cities and live in hiding, scraping by.
-Each of 18 human cities is guarded by Resistance soldiers, which serve as defence for people from foreign gods invasion.
+It is a big country in the eastern part of the continent, and at the moment it was under control of the foreign gods.
+Humans, who lost in the war, had to reconstruct ruins into cities and live in hiding, somehow scraping by.
+Each of the 18 human cities is guarded by Resistance soldiers, who serve as defence for the people from the foreign gods invasion.
 
-Headquarters of the Resistance was in 8th ward Cassiopeia.
-It was a site of once large scale factory, and in one of the areas there...
+The headquarters of the Resistance was in 8th ward Cassiopeia.
+It was at the site of what was once a large scale factory, and in one of the areas there...
 
 "Jeanne-dono, are you about to depart...?"
```

Comment:
I need to check original and re-phrase it.
There should talking about Jeanne being about to depart